samples/highcharts/data/livedata-rows/demo.js

```diff
@@ -0,0 +1,19 @@
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'spline',
+        height: 800
+    },
+    title: {
+        text: 'Live Data (Rows JSON)'
+    },
+
+    subtitle: {
+        text: 'Data input from a remote JSON file'
+    },
+
+    data: {
+        rowsURL: 'https://demo-live-data.highcharts.com/time-rows.json',
```

Comment:
Like with livedata-columns, the first row containing series names is missing, so the series name keeps changing with the data.